test_rclcpp/test/test_client_scope_consistency_server.cpp

```diff
@@ -39,9 +39,13 @@ int main(int argc, char ** argv)
     "client_scope", handle_add_two_ints, rmw_qos_profile);
 
   rclcpp::WallRate loop_rate(30);
-  while (rclcpp::ok()) {
-    rclcpp::spin_some(node);
-    loop_rate.sleep();
+  try {
+    while (rclcpp::ok()) {
+      rclcpp::spin_some(node);
+      loop_rate.sleep();
+    }
+  } catch (const rclcpp::exceptions::RCLError & ex) {
```

Comment:
As discussed in other places, we should really fix this issue so this "catch all rcl errors" thing isn't needed (this mechanism can be masking actual errors).

Reply:
I agree. I think this is the most relevant open issue at the moment: https://github.com/ros2/rclcpp/issues/1139